### PR TITLE
Revamp commercial handlers

### PR DESCRIPTION
**Pull request by Mauzis01**

### CODE REVIEW

File: src/models/index.ts [hunk 1]
```diff
@@ -1,9 +1,4 @@
-export * from "../internal/enums/ratios";
-export * from "../internal/enums/statuses";
-export * from "../internal/enums/styles";
-export * from "../internal/enums/controls";
-export * from "../internal/types/generations";
-export * from "../internal/types/inpaint";
-export * from "../internal/types/remix";
-export * from "../internal/types/variations";
+export * from "../internal/enums";
+
```
Comment (m-shariq) [inline src/models/index.ts]: Remove extra line

File: src/internal/types/config.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+ *
+ * @property {string} [negativePrompt] - The negative prompt for contrasting images.
+ * @property {number} [seed] - The random seed for reproducible generation.
+ */
+export declare type AIClothes = {
+  negativePrompt?: string;
+  seed?: number;
+};
+
+/**
```
Comment (m-shariq) [inline src/internal/types/config.ts]: Keep all in same folder structure as we have in handlers

File: src/internal/enums/filters.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  VAN_GOUH = 1,
+  POP_ART = 2,
+  CURE_CHIBI = 3,
+  CRACKED_KINTSUGI = 4,
+  NEON_ME = 5,
+  SIMPLE_CARTOON = 6,
+  WESTERN_ANIME = 7,
+  "80s_ANIME" = 8,
+  NEWSPAPER_RETRO = 9,
+}
```
Comment (m-shariq) [inline src/internal/enums/filters.ts]: It should be Filters only.

File: src/internal/handlers/edits/filters.ts [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  return {
+    finalFilterId,
+    finalStyleId,
+  };
+};
+
+const filters = async (
+  client: RequestClient,
+  image: ImageParam,
+  filterId: AIFilters
```
Comment (m-shariq) [inline src/internal/handlers/edits/filters.ts]: Here must be a config

File: src/internal/handlers/edits/filters.ts [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+import { FormDataBuilder } from "src/internal/utils/form";
+
+const getIds = (filterId: AIFilters) => {
+  const finalFilterId = filterId === AIFilters.PIXAR_DISNEY ? 1 : filterId;
+  const finalStyleId = filterId === AIFilters.PIXAR_DISNEY ? 2 : 1;
+  return {
+    finalFilterId,
+    finalStyleId,
+  };
+};
```
Comment (m-shariq) [inline src/internal/handlers/edits/filters.ts]:
```
const ids = (id: AIFilters) => ({
  filterId: id === AIFilters.PIXAR_DISNEY ? AIFilters.PIXAR_DISNEY : id,
  styleId: id === AIFilters.PIXAR_DISNEY ? AIFiltersStyle.style2 : AIFiltersStyle.style1,
});
```

**Review by m-shariq — CHANGES_REQUESTED** [review]

File: dist/internal/types/filters.d.ts [hunk 6]
```diff
@@ -0,0 +1,9 @@
+import { Filters } from "../enums";
+/**
+ * Configuration options for AI Filters.
+ *
+ * @property {Filters} [filterId] - The filter id for AI Filters application (model: Filters).
+ */
+export declare type AIFitlersConfig = {
```
Comment (m-shariq) [inline dist/internal/types/filters.d.ts]: Why it's AIFitlers?

**Review by m-shariq — CHANGES_REQUESTED** [review]

File: dist/internal/types/filters.d.ts (hunk 6 above)
Comment (Mauzis01) [reply dist/internal/types/filters.d.ts]: Was a typo. Fixed that.

**Pull request closed without merging**